dist/AureliaPlugin.js

```diff
@@ -1,146 +1,146 @@
-        // with aurelia-loader, while still enabling tree shaking all other exports.
-        new PreserveExportsPlugin_1.PreserveExportsPlugin());
-    }
-}
-exports.AureliaPlugin = AureliaPlugin;
-;
-function getEntry(entry) {
-    // Fix: ideally we would require `entry` to be a string
-    //      but in practice, using webpack-dev-server might shift one (or two --hot) extra entries.
-    if (typeof entry === "object")
-        entry = entry[Object.getOwnPropertyNames(entry)[0]];
-    if (Array.isArray(entry))
-        entry = entry[entry.length - 1];
-    if (typeof entry !== "string")
-        throw new Error("includeAll option only works with a single entry point.");
-    return entry;
-}
-function getPAL(target) {
-    switch (target) {
-        case "web": return "aurelia-pal-browser";
-        case "webworker": return "aurelia-pal-worker";
-        default: return "aurelia-pal-node";
-    }
-}
-const configModules = {};
-let configModuleNames = {
-    "defaultBindingLanguage": "aurelia-templating-binding",
-    "router": "aurelia-templating-router",
-    "history": "aurelia-history-browser",
-    "defaultResources": "aurelia-templating-resources",
-    "eventAggregator": "aurelia-event-aggregator",
-    "developmentLogging": "aurelia-logging-console",
-};
-// "configure" is the only method used by .plugin()
-for (let c in configModuleNames)
-    configModules[c] = { name: configModuleNames[c], exports: ["configure"] };
-// developmentLogging has a pre-task that uses ConsoleAppender
-configModules['developmentLogging'].exports.push("ConsoleAppender");
-function getConfigModules(config) {
-    if (!config)
-        return undefined;
-    if (!Array.isArray(config))
-        config = [config];
-    // Expand "standard"
-    let i = config.indexOf("standard");
-    if (i >= 0)
-        config.splice(i, 1, "basic", "history", "router");
-    // Expand "basic"
-    i = config.indexOf("basic");
-    if (i >= 0)
-        config.splice(i, 1, "defaultBindingLanguage", "defaultResources", "eventAggregator");
-    return config.map(c => configModules[c]);
-}
+"use strict";
+const AureliaDependenciesPlugin_1 = require("./AureliaDependenciesPlugin");
+const ConventionDependenciesPlugin_1 = require("./ConventionDependenciesPlugin");
+const DistPlugin_1 = require("./DistPlugin");
+const GlobDependenciesPlugin_1 = require("./GlobDependenciesPlugin");
+const HtmlDependenciesPlugin_1 = require("./HtmlDependenciesPlugin");
+const ModuleDependenciesPlugin_1 = require("./ModuleDependenciesPlugin");
+const PreserveExportsPlugin_1 = require("./PreserveExportsPlugin");
+const PreserveModuleNamePlugin_1 = require("./PreserveModuleNamePlugin");
+const SubFolderPlugin_1 = require("./SubFolderPlugin");
+class AureliaPlugin {
+    constructor(options = {}) {
+        this.options = Object.assign({
+            includeAll: false,
+            aureliaApp: "main",
+            aureliaConfig: ["standard", "developmentLogging"],
+            dist: "native-modules",

```

Comment:
I'm thinking if beside a string this could be an array of possibilities. 

The use-case is that sometimes the folder `native-modules` might not be available, and we'd like to fallback to a different folder (e.g. `commonjs`). Or, we'd like to use `es2017` (as present in [some packages](https://github.com/aurelia/loader-webpack/tree/master/dist)) and fall-back to `native-modules`. This is useful when developing plugins that use `async/await` and you want to test and debug them in Chrome or Edge, which already support `async/await` natively.